s3watcher/lambda/lib/Transfer.js

```diff
@@ -23,25 +22,24 @@ module.exports = {
         const upload = Upload.buildUpload(config, s3Event);
 
         const success = function() {
-            Logger.logDelete(config.stage, s3Object);
-
+            logger.log('Deleting from ingest bucket.', s3Object);
             return S3Helper.deleteS3Object(s3Object);
         };
 
         const failGraceful = function(e) {
-            Logger.logImportFail(config.stage, e);
-            Logger.logCopyToFailBucket(config.stage, s3CopyObject);
+            logger.log('Import failed.', e);
```

Comment:
😍 method chaining is so nice! Want to raise a PR in the logger module? (I can do it tomorrow if not).